docs/zh/latest/plugins/response-rewrite.md

```diff
@@ -125,7 +125,7 @@ X-Server-balancer_addr: 127.0.0.1:80
 
 如果你在 `access` 阶段执行了 `ngx.exit`，该操作只是中断了请求处理阶段，响应阶段仍然会处理。如果你配置了 `response-rewrite` 插件，它会强制覆盖你的响应信息（如响应代码）。
 
-![ngx.edit tabular overview](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png)
+![ngx.exit tabular overview](https://user-images.githubusercontent.com/2106987/170849580-aa296d3d-b43a-404a-8c88-720afb2b7c08.png)
```

Comment:
This picture is a table, can we turn it into a table? Like the table under properties.

Reply:
Good idea!

Reply:
This way is more miantainable.

Reply:
Ok, I'll change with plaintext table.

Reply:
Thanks! cc @hf400159 to have a check 🙏